dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -902,6 +899,7 @@ void Aggregator::spill(AggregatedDataVariants & data_variants)
     data_variants.aggregates_pools = Arenas(1, std::make_shared<Arena>());
     data_variants.aggregates_pool = data_variants.aggregates_pools.back().get();
     data_variants.without_key = nullptr;
+    agg_spill_context->updatePerThreadRevocableMemory(data_variants.revocableBytes(), thread_num);
```

Comment:
Is it useful here? What happens when true is returned?

Reply:
Removed.